docs/admin/kubeadm.md

```diff
@@ -4,1010 +4,22 @@ approvers:
 - luxas
 - errordeveloper
 - jbeda
-title: Kubeadm
+title: Overview of Kubeadm
 notitle: true
 ---
-# kubeadm Setup Tool Reference Guide
+# Overview of Kubeadm
 
```

Comment:
To be consistent with other parts of the documentation, I would suggest this convention: Make kubeadm lower-case unless it is the first word in a sentence.

So in titles, headings, and sentences (except first word), make kubeadm lower-case.